packages/@glimmer-workspace/integration-tests/lib/modifiers.ts

```diff
@@ -44,7 +44,7 @@ export class TestModifierManager
     return tag;
   }
 
-  getDebugName({ Klass }: TestModifierDefinitionState) {
+  getDebugName(state: TestModifierDefinitionState, { Klass }: TestModifierDefinitionState) {
```

Comment:
This change seems fishy, why is it passing the same argument twice?

Reply:
the type was the same, but not the object. anyway it was confusing, i changed it